backend/test/features/object-article.feature

```diff
@@ -5,26 +5,96 @@ Feature: Send and receive Articles
     Given our own server runs at "http://localhost:4123"
     And we have the following users in our database:
       | Slug         |
-      | marvin       |
-      | max          |
+      | hans         |
+      | gerald       |
 
-  Scenario: Send an article to a user inbox and make sure it's added to the inbox
-    When I send a POST request with the following activity to "/activitypub/users/max/inbox":
+  Scenario: Send an article to the shared inbox and make sure it's added to the correct inbox
+    When I send a POST request with the following activity to "/inbox":
     """
     {
       "@context": "https://www.w3.org/ns/activitystreams",
-      "id": "https://aronda.org/users/marvin/status/lka7dfzkjn2398hsfd",
+      "id": "https://localhost:4123/api/users/gerald/status/lka7dfzafdgkjnasg2398hsfd",
       "type": "Create",
-      "actor": "https://aronda.org/users/marvin",
+      "actor": "http://localhost:4123/users/gerald",
       "object": {
-          "id": "https://aronda.org/users/marvin/status/kljsdfg9843jknsdf",
-          "type": "Article",
-          "published": "2019-02-07T19:37:55.002Z",
-          "attributedTo": "https://aronda.org/users/marvin",
-          "content": "Hi Max, how are you?",
-          "to": "as:Public"
+        "id": "https://localhost:4123/api/users/gerald/status/kljsasddfg9843jknsdfasd",
+        "type": "Article",
+        "published": "2019-02-07T19:37:55.002Z",
+        "attributedTo": "https://localhost:4123/api/users/gerald",
+        "content": "Content for article object creation!",
+        "to": "as:Public"
       }
     }
     """
     Then I expect the status code to be 200
-    And the post with id "kljsdfg9843jknsdf" to be created
+    And the post with id "kljsasddfg9843jknsdfasd" to be created
+
+  Scenario: Send an note to the shared inbox and make sure it's added to the correct inbox
```

Comment:
Do we map a `Note` and an `Article` to a Post?

Reply:
Yes both are mapped to a Post.